src/Connectors/test/Connector.EFCore.Test/EntityFrameworkCoreTypeLocatorTest.cs

```diff
@@ -60,4 +60,30 @@ public void Options_Found_In_Devart_Assembly()
         Assert.NotNull(type);
         EntityFrameworkCoreTypeLocator.OracleEntityAssemblies = oracleAssemblies;
     }
+
+    [Fact]
+    public void Options_Found_In_Pomelo_Assembly()
+    {
+        // arrange ~ narrow the assembly list to one specific nuget package
+        var types = EntityFrameworkCoreTypeLocator.MySqlEntityAssemblies;
+        EntityFrameworkCoreTypeLocator.MySqlEntityAssemblies = new[] { "Pomelo.EntityFrameworkCore.MySql" };
+
+        var type = EntityFrameworkCoreTypeLocator.MySqlDbContextOptionsType;
+
+        Assert.NotNull(type);
+        EntityFrameworkCoreTypeLocator.MySqlEntityAssemblies = types;
+    }
+
+    [Fact(Skip = "Change NuGet reference to see this test pass")]
+    public void Options_Found_In_MySql_Assembly()
+    {
+        // arrange ~ narrow the assembly list to one specific nuget package
+        var types = EntityFrameworkCoreTypeLocator.MySqlEntityAssemblies;
+        EntityFrameworkCoreTypeLocator.MySqlEntityAssemblies = new[] { "MySql.EntityFrameworkCore" };
+
+        var type = EntityFrameworkCoreTypeLocator.MySqlDbContextOptionsType;
+
+        Assert.NotNull(type);
+        EntityFrameworkCoreTypeLocator.MySqlEntityAssemblies = types;
+    }
```

Comment:
This test used to be netcore31 only. I inferred that we switched to a different third-party package for net60+, but looking at this, my assumption may be incorrect. Is this something that should work on net60 as well?

Reply:
IIRC there are at least 2 packages that can be used and they worked in the same scenarios. We were just using framework to be able to test both of those packages

Reply:
Makes sense, we should keep it then.